share/picongpu/tests/XrayScattering/include/picongpu/param/species.param

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2014-2020 Rene Widera, Richard Pausch, Pawel Ordyna
```

Comment:
why is Annegret listed here as co-author 
(applies to various other files as well)

Reply:
She implemented the Higuera-Cary particle pusher which can be selected in the `species.param`. Since she edited this file to make it available she is an author. And since I copied the new standard file to all examples, she is now an author in several files were she hasn't been an author before.